src/lifecycle/paths.h

```diff
@@ -1,36 +1,33 @@
 #ifndef GODOT_JVM_PATHS_H
 #define GODOT_JVM_PATHS_H
 
-static constexpr const char* DESKTOP_BOOTSTRAP_FILE {"godot-bootstrap.jar"};
-static constexpr const char* DESKTOP_USER_CODE_FILE {"main.jar"};
+static constexpr const char* DESKTOP_BOOTSTRAP_FILE {"jvm/godot-bootstrap.jar"};
```

Comment:
We have the folder `jvm` as a separate constant for the kotlin code already. Can we do this for the cpp code as well please? Then it would be consistent and easy to change.

Reply:
as discussed on discord; not worth the effort